v1/api/swagger/components/schemas/Project.yaml

```diff
@@ -37,3 +37,6 @@ allOf:
         example:
           - 046b6c7f-0b8a-43b9-b35d-6489e6daee91
           - 91eec8b5-6460-4532-aad1-47b983edb63c
+      ReplicationEnabled:
```

Comment:
should this be a required field since this is for showing the current value (true or false)?

Reply:
no, required is for create